spec/comment_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Comments, type: :model do
+  before(:all) do
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.',
+                        posts_counter: 0)
+    @post = Post.create(author: @user, title: 'Post communication', text: 'This is my first post', likes_counter: 0,
+                        comments_counter: 0)
+  end
+
+  context 'Associations' do
+    it 'belongs to an author' do
+      comment = Comments.reflect_on_association('author')
```

Comment:
- [x] Same case as above.

```suggestion
      comment = Comment.reflect_on_association('author')
```